tests/projects/factories.py

```diff
@@ -20,3 +20,16 @@ class Meta:
     creator = factory.SubFactory(a4_factories.USER_FACTORY)
     project = factory.SubFactory(a4_factories.ProjectFactory)
     email = factory.Sequence(lambda n: "user%d@liqd.net" % n)
+
+
+class ProjectInsightFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.ProjectInsight
+
+    project = factory.SubFactory(a4_factories.ProjectFactory)
+    comments = 8
+    ratings = 4
+    written_ideas = 3
+    poll_answers = 2
+    live_questions = 2
+    display = True
```

Comment:
suggestion: add 
```
    @factory.post_generation
    def active_participants(self, create, extracted, **kwargs):
        if not create or not extracted:
            return
        self.active_participants.add(*extracted)
```
this way we can pass users like this: `project_insight_factory.create(project=project, active_participants=[user, user1,....])`

Reply:
thanks!